docs/standard/library-guidance/dependencies.md

```diff
@@ -62,17 +62,17 @@ For more information, see [Package versioning](/nuget/concepts/package-versionin
 
 One way to reduce external NuGet package dependencies is to reference shared source packages. A shared source package contains [source code files](/nuget/reference/nuspec#including-content-files) that are included in a project when referenced. Because you're just including source code files that are compiled with the rest of your project, there's no external dependency and chance of conflict.
 
-Shared source packages are great for including small pieces of functionality. For example, a shared source package of helper methods for making HTTP calls.
+Shared source packages are great for including small pieces of functionality. For example, you might reference a shared source package of helper methods for making HTTP calls.
 
-![Shared source package](./media/dependencies/shared-source-package.png "Shared source package")
+![Shared source package](./media/dependencies/shared-source-package.png)
 
 ```xml
 <PackageReference Include="Microsoft.Extensions.Buffers.Testing.Sources" PrivateAssets="All" Version="1.0" />
 ```
 
-![Shared source project](./media/dependencies/shared-source-project.png "Shared source project")
+![Shared source project](./media/dependencies/shared-source-project.png)
 
-Shared source packages have some limitations. They can only be referenced by `PackageReference`, so older `packages.config` projects are excluded. Also shared source packages are only usable by projects with the same language type. Because of these limitations shared source packages are best used to share functionality within an open-source project.
+Shared source packages have some limitations. They can only be referenced by `PackageReference`, so older `packages.config` projects are excluded. Also, shared source packages are only usable by projects with the same language type. Because of these limitations, shared source packages are best used to share functionality within an open-source project.
```

Comment:
```suggestion
Shared source packages have some limitations. They can only be referenced by `PackageReference`, so older `packages.config` projects are excluded. Also, shared source packages are only usable by projects with the same language. Because of these limitations, shared source packages are best used to share functionality within an open-source project.
```